backend/reviews/old-xml-app-store-scraper/README.md

```diff
@@ -0,0 +1,289 @@
+# app-store-scraper [![Build Status](https://secure.travis-ci.org/facundoolano/app-store-scraper.png)](http://travis-ci.org/facundoolano/app-store-scraper)
```

Comment:
DIdn't we remove the old scraper files before? Is there a reason we still need them?